pallets/gear/src/lib.rs

```diff
@@ -405,25 +405,25 @@ pub mod pallet {
         ProgramAlreadyExists,
         /// Program is terminated.
         ///
-        /// Program init ended up with failure, so such message destination is unavailable anymore.
+        /// Program init failed, so such message destination is no longer unavailable.
         InactiveProgram,
         /// Message gas tree is not found.
         ///
-        /// When message claimed from mailbox has a corrupted or non-extant gas tree associated.
+        /// When a message claimed from the mailbox has a corrupted or non-extant gas tree associated.
         NoMessageTree,
         /// Code already exists.
         ///
-        /// Occurs when trying to save to storage a program code, that has been saved there.
+        /// Occurs when trying to save to storage a program code that has been saved there.
         CodeAlreadyExists,
-        /// Code not exists.
+        /// Code does not exist.
         ///
         /// Occurs when trying to get a program code from storage, that doesn't exist.
-        CodeNotExists,
+        CodeDoesntExist,
         /// The code supplied to `upload_code` or `upload_program` exceeds the limit specified in the
         /// current schedule.
         CodeTooLarge,
         /// Failed to create a program.
-        FailedToConstructProgram,
+        ProgramConstructFailed,
```

Comment:
Perhaps `ProgramConstructionFailed` is better? What do you think @AndrePanin @breathx @shamilsan ?

Reply:
Agree. Or even `ProgramCreationFailed`.